tgui/packages/tgui/interfaces/ClothingBooth/GroupingTag.tsx

```diff
@@ -0,0 +1,55 @@
+import { Box, Stack } from '../../components';
+import { buildFieldComparator, numberComparator } from './utils/comparator';
+import { ClothingBoothData, ClothingBoothGroupingTagsData, ClothingBoothSlotKey } from './type';
+import { useBackend } from '../../backend';
+
+interface GroupingTagContainerProps {
+  slot: ClothingBoothSlotKey;
+  grouping_tags: string[];
+}
+
+interface ClothingBoothSlotDetail {
+  id: ClothingBoothSlotKey;
+  name: string;
+}
+
+export const GroupingTagContainer = (props: GroupingTagContainerProps, context) => {
```

Comment:
Prefer `GroupingTags` as a name here. This would make the file as a whole `GroupingTags.tsx`.